docs/http.md

```diff
@@ -0,0 +1,61 @@
+# HTTP API
+
+When a Cog Docker image is run, it serves an HTTP API for making predictions.
+
+First, build your model:
+
+    cog build -t my-model
+
+Then, start the Docker container:
+
+    docker run -d -p 5000:5000 --init my-model
```

Comment:
Is `--init` needed here? And if so, why?

I read the [docker docs on `--init`](https://docs.docker.com/engine/reference/run/), but still don't really get what this is about.

> You can use the `--init` flag to indicate that an init process should be used as the PID 1 in the container. Specifying an init process ensures the usual responsibilities of an init system, such as reaping zombie processes, are performed inside the created container.
https://www.loom.com/share/41790390b7f1435c980bd5adddce7ff8

Reply:
https://www.loom.com/share/41790390b7f1435c980bd5adddce7ff8

Reply:
It's so signals work. Ctrl-C, etc.

Reply:
Zeke says this is fixed in FastAPI now!